portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/site/SitesExportImport.testcase

```diff
@@ -32,6 +32,73 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-128151. Export and import a site containing a content page into the default site."
+	@priority = "4"
+	test ExportImportSiteIntoDefaultSite {
+		property portal.acceptance = "true";
+
+		task ("Add a Content Page via JSON") {
+			JSONLayout.addPublicLayout(
+				groupName = "Test Site Name",
+				layoutName = "Test Content Page Name",
+				type = "content");
+		}
+
+		task ("Publish the content page") {
+			ContentPagesNavigator.openEditContentPage(
+				pageName = "Test Content Page Name",
+				siteName = "Test Site Name");
+
+			PageEditor.clickPublish();
+		}
+
+		task ("Assert the friendly URL") {
+			SitePages.openPagesAdmin(siteURLKey = "test-site-name");
+
+			SitePages.gotoPageMenuItem(
+				menuItem = "Configure",
+				pageName = "Test Content Page Name");
+
+			Page.gotoFriendlyURLHistory();
+
+			Page.viewActiveURL(activeURL = "/test-content-page-name");
+		}
+
+		task ("Export the created test site") {
+			LAR.exportSiteCP(siteScopeName = "Test Site Name");
+
+			var key_larFileName = "${larFileName}";
+
+			var larFileName = LAR.getLarFileName();
+
+			LAR.downloadLar();
+		}
+
+		task ("Import the site into the default site") {
+			LAR.importSiteCP(
+				larFileName = "${larFileName}",
+				siteName = "guest");
+		}
+
+		task ("Assert the friendly URL into Default Site") {
+			SitePages.openPagesAdmin(siteURLKey = "guest");
+
+			SitePages.gotoPageMenuItem(
+				menuItem = "Configure",
+				pageName = "Test Content Page Name");
+
+			Page.gotoFriendlyURLHistory();
+
+			Page.viewActiveURL(activeURL = "/test-content-page-name");
+		}
+
+		task ("Delete the content page in the default site") {
+			JSONLayout.deletePublicLayout(
+				groupName = "Guest",
+				layoutName = "Test Content Page Name");
+		}
```

Comment:
Unneeded

Reply:
Hi @Tim-Cao, just to double confirm, if I remove this step it'll let the content page in the default site, is it ok? the teardown isn't excluding it.